app/wallet/Main.hs

```diff
@@ -0,0 +1,126 @@
+{-# LANGUAGE DuplicateRecordFields #-}
+{-# LANGUAGE QuasiQuotes #-}
+
+module Main where
```

Comment:
Side-note but this cli and `cardano-wallet-server` are one and one thing. We do want to have them merged together in the end, so that there's one CLI to spawn the server, and the same one to interact with it :+1:

Reply:
> Side-note but this cli and `cardano-wallet-server` are one and one thing. We do want to have them merged together in the end, so that there's one CLI to spawn the server, and the same one to interact with it +1

Understood! In that case, are you expecting something like this:

`cardano-wallet server start ...`
`cardano-wallet wallet create --name wibble ...`
`cardano-wallet wallet list`